### PR TITLE
#[161709558] Added endpoints to get deliveries by one user

### PR DESCRIPTION
**Pull request by MandelaK**
#### What does this PR do?
We added endpoints allowing users to get all deliveries by one user

#### Description of Task to be completed?
We need endpoints that allow us to get all deliveries by one user. For this we gave each user a unique user_id that we then use to retrieve the orders made by the user.

#### How should this be manually tested?
After cloning the repo and installing all dependencies, you can head over to the endpoint at GET /api/v1/users/<int:user_id>/parcels to get all deliveries made by the user. You will get all the deliveries made by the user, or an error message in case none are found.

#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
#161709558

### CODE REVIEW

File: app/api/v1/models.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# This is our models file where we store our temporary storage data and methods
+# to manipulate the data. Our parcels list is globally available all classes
+# that inherit the Parcel class
+
+# this is where all our parcels will be appended
+parcels = [
+    {
+        "id": 1,
```
Comment (cosmas28) [inline app/api/v1/models.py:8]: Kindly be persistent when using double quotes. I noticed you mix both single and double quotes, so please use one of them in every file.

File: tests/v1/test_views.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        """This should be called before each test"""
+        self.app = app
+        self.client = app.test_client(self)
+        self.app.testing = True
+        self.parcel = parcel
+        self.app_context = self.app.app_context()
+        self.app_context.push()
+
+
+class TestValidRequest(ParcelTestCase):
```
Comment (cosmas28) [inline tests/v1/test_views.py:32]: Write more tests, don't forget edge cases.

**Review by cosmas28 — CHANGES_REQUESTED** [review]
